airbyte-cdk/python/unit_tests/sources/file_based/scenarios/csv_scenarios.py

```diff
@@ -19,7 +19,7 @@
                     "validation_policy": "emit_record",
                 }
             ],
-            "start_date": "2023-06-04T03:54:07Z"
+            "start_date": "2023-06-04T03:54:07Z",
```

Comment:
This file looks crazy, but I ran formatting which changes a ton of stuff which might make this a bit hard to review.  To summarize the actual changes I made, I did some reordering of the file formats and adding `AvroFormat` which had to then be added to the scenario used to verify Spec. The rest remained mostly the same